Cvc5Test/Init.lean

```diff
@@ -0,0 +1,35 @@
+import cvc5
+
+namespace cvc5
+
+def Test.assertEq [ToString α] [BEq α] (lft rgt : α) (hint := "") : IO Unit := do
+  if lft != rgt then
+    let pref := if hint.isEmpty then "" else s!"[{hint}] "
+    IO.eprintln s!"{pref}comparison failed: `{lft}` is different from `{rgt}`"
+
+def Test.assertNe [ToString α] [BEq α] (lft rgt : α) (hint := "") : IO Unit := do
+  if lft == rgt then
+    let pref := if hint.isEmpty then "" else s!"[{hint}] "
+    IO.eprintln s!"{pref}comparison failed: `{lft}` is the same as `{rgt}`"
```

Comment:
Assertion failures should cause execution to halt. Consider throwing an exception. Eventually, we create a Test monad (for solver initialization).

Reply:
Now using `IO.throwServerError`.